PERSONA.md

```diff
@@ -0,0 +1,18 @@
+# Persona
+
```

Comment:
Could we have a table to summarize the personas?

Example:

| Persona | Docker | Docker Compose | Linux |
|:--------:|:---------:|:--------------------:|:-----:|
| First | No or Little Knowledge | No or Little Knowledge | No | 
| Second | Have Knowledge | Have Knowledge | Have |

It doesn't need to map all tools or concepts.